_automation_cookbook/append-array-list.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Appending to Arrays and Lists
+excerpt: Learn different methods for appending values to arrays and lists in Cerb automations.
```

Comment:
Just for clarity, the `excerpt:` field shows in search engine results, but we also use it on the index of the cookbook. So it works best in the imperative form where it's like a tooltip on a magic button, and we just say "Append values to an array or list". 👍🏻